index.html

```diff
@@ -1,115 +1,74 @@
-    "mark": "circle",
-    "encoding": {
-      "x": {"field": "Horsepower", "type": "quantitative"},
-      "y": {"field": "Miles_per_Gallon", "type": "quantitative"}
-    }};
-  var scatterEmbed = {
-    mode: "vega-lite",  // Instruct Vega-Embed to use the Vega-Lite compiler
-    spec: scatterSpec
-  };
-  vg.embed("#vis-scatter", scatterEmbed, function(error, result) {
-    addTooltip(result.view);
-  });
-
-  // Stacked Bar Chart
-  var stackedbarSpec = {
-    "data": {"url": "data/seattle-weather.csv", "formatType": "csv"},
-    "mark": "bar",
-    "encoding": {
-      "x": {
-        "field": "date",
-        "type": "temporal",
-        "timeUnit": "month",
-        "axis": {"title": "Month of the year"}
-      },
-      "y": {
-        "field": "*",
-        "type": "quantitative",
-        "aggregate": "count"
-      },
-      "color": {
-        "field": "weather",
-        "type": "nominal",
-        "scale": {
-          "domain": ["sun","fog","drizzle","rain","snow"],
-          "range": ["#e7ba52","#c7c7c7","#aec7e8","#1f77b4","#9467bd"]
-        },
-        "legend": {
-          "title": "Weather type"
-        }
-      }
-    }
-  };
-  var stackedbarEmbed = {
-    mode: "vega-lite",
-    spec: stackedbarSpec
-  };
-  vg.embed("#vis-stacked-bar", stackedbarEmbed, function(error, result) {
-    addTooltip(result.view);
-  });
-
-  // Simple Bar Chart
-  var barSpec = {
-    "description": "A simple bar chart with embedded data.",
-    "data": {
-      "values": [
-        {"a": "A","b": 28}, {"a": "B","b": 55}, {"a": "C","b": 43},
-        {"a": "D","b": 91}, {"a": "E","b": 81}, {"a": "F","b": 53},
-        {"a": "G","b": 19}, {"a": "H","b": 87}, {"a": "I","b": 52}
-      ]
-    },
-    "mark": "bar",
-    "encoding": {
-      "x": {"field": "a", "type": "ordinal"},
-      "y": {"field": "b", "type": "quantitative"}
-    }
-  };
-  var barEmbed = {
-    mode: "vega-lite",
-    spec: barSpec
-  }
-  vg.embed("#vis-bar", barEmbed, function(error, result) {
-    addTooltip(result.view);
-  })
-
-  </script>
+  <!-- Example Page -->
+  <script src="examples.js"></script>
+  <link rel="stylesheet" type="text/css" href="examples.css">
+
 </head>
+
 <body>
-  <!-- Scatter Plot Tooltip Example -->
+  <h1>Vega-Lite Tooltip Examples</h1>
+
+  <!-- Scatter Plot -->
   <div id="vis-scatter" class="tooltip-example"></div>
 
-  <!-- Stacked Bar Chart Tooltip Example -->
-  <div id="vis-stacked-bar" class="tooltip-example"></div>
+  <!-- Bubble Plot -->
+  <div id="vis-bubble-multi-aggr" class="tooltip-example"></div>
 
-  <!-- Simple Bar Chart Example -->
+  <!-- Trellis Barley -->
+  <div id="vis-trellis-barley" class="tooltip-example"></div>
+
+
+  <!-- Simple Bar Chart -->
   <div id="vis-bar" class="tooltip-example"></div>
 
+  <!-- Stacked Bar Chart -->
+  <div id="vis-stacked-bar" class="tooltip-example"></div>
+
+  <!-- Layered Bar Chart -->
+  <div id="vis-layered-bar" class="tooltip-example"></div>
+
+
+  <!-- Line Chart -->
```

Comment:
Add note for "still-broken" examples?  I remember line and area do not behave correctly yet?